docs/source/notes/autograd.rst

```diff
@@ -36,6 +36,52 @@ flow statements, that can change the overall shape and size of the graph at
 every iteration. You don't have to encode all possible paths before you
 launch the training - what you run is what you differentiate.
 
+.. _saved-tensors-doc:
+
+Saved tensors
+^^^^^^^^^^^^^
+
+Some operations need intermediary results to be saved during the forward pass
+in order to execute the backward pass. For example, the function
+:math:`x\mapsto x^2` needs to save the input :math:`x` because it is needed to
+compute the gradient.
+
+When defining a custom Python :class:`~torch.autograd.Function`, you can use
+:func:`~torch.autograd.function._ContextMethodMixin.save_for_backward` to save
+tensors during the forward pass and
+:attr:`~torch.autograd.function.Function.saved_tensors` to retrieve them
+during the backward pass. See :doc:`/notes/extending` for more information.
+
+For operations that Pytorch defines (e.g. ``aten::pow``), tensors are
+automatically saved as they are needed. You can explore which tensors are saved
+by a certain ``grad_fn`` by looking for its attributes starting with the prefix
+``_saved``.
+
+.. code::
+
+    x = torch.randn(5, requires_grad=True)
+    y = x.pow(2)
+    print(x.equal(y.grad_fn._saved_self)) # True
+    print(x is y.grad_fn._saved_self) # True
```

Comment:
Nit: Python style guide says two spaces have to exist before the start of an inline comment